server/dashboard.js

```diff
@@ -443,7 +443,7 @@ async function submitNewClient(req, res) {
         true,
         false,
         false,
-        'en',
+        req.body.language || 'en',
```

Comment:
Looks good! 
Small note, `req.body` is stored in the variable `data` on line 411, so you can update this line to be `data.language || 'en',` to be more consistent with the rest of the arguments.

Reply:
Thanks for the comment, I have modified the code to use the data constant instead!